New physics overrides (#11465)
minetest/minetest: doc/lua_api.md

        * 9 - zoom
    * Returns `0` (no bits set) if the object is not a player.
* `set_physics_override(override_table)`
    * Overrides the physics attributes of the player
    * `override_table` is a table with the following fields:
        * `speed`: multiplier to default movement speed value (default: `1`)
        * `jump`: multiplier to default jump value (default: `1`)
        * `gravity`: multiplier to default gravity value (default: `1`)
        * `speed_climb`: multiplier to default climb speed value (default: `1`)
            * Note: The actual climb speed is the product of `speed` and `speed_climb`
        * `speed_crouch`: multiplier to default sneak speed value (default: `1`)
            * Note: The actual sneak speed is the product of `speed` and `speed_crouch`
        * `liquid_fluidity`: multiplier to liquid movement resistance value
          (for nodes with `liquid_move_physics`); the higher this value, the lower the
          resistance to movement. At `math.huge`, the resistance is zero and you can
          move through any liquid like air. (default: `1`)

>           Warning: Values below 1 are currently unsupported.

Reviewer: Why? Because of the division by zero below? If there is a limit, it should be enforced in the code too (e.g. `MYMAX`).

        * 9 - zoom
    * Returns `0` (no bits set) if the object is not a player.
* `set_physics_override(override_table)`
    * Overrides the physics attributes of the player
    * `override_table` is a table with the following fields:

>         * `speed`: multiplier to default movement speed value (default: `1`)

Reviewer 2: ```suggestion
        * `speed`: multiplier to default movement speed and acceleration values (default: `1`)
```

            * Note: The actual climb speed is the product of `speed` and `speed_climb`
        * `speed_crouch`: multiplier to default sneak speed value (default: `1`)
            * Note: The actual sneak speed is the product of `speed` and `speed_crouch`
        * `liquid_fluidity`: multiplier to liquid movement resistance value
          (for nodes with `liquid_move_physics`); the higher this value, the lower the
          resistance to movement. At `math.huge`, the resistance is zero and you can
          move through any liquid like air. (default: `1`)
          Warning: Values below 1 are currently unsupported.
        * `liquid_fluidity_smooth`: multiplier to default maximum liquid resistance value
          (for nodes with `liquid_move_physics`); controls deceleration when entering
          node at high speed. At higher values you come to a halt more quickly
          (default: `1`)
        * `liquid_sink`: multiplier to default liquid sinking speed value;
          (for nodes with `liquid_move_physics`) (default: `1`)
        * `acceleration_default`: multiplier to horizontal and vertical acceleration

>           on ground or when climbing (default: `1`)

Reviewer 2: ```suggestion
          on ground or when climbing (default: `1`)
            * Note: The actual acceleration is the product of `speed` and `acceleration_default`
```

        * `acceleration_air`: multiplier to acceleration

>           when jumping or falling (default: `1`)

Reviewer 2: ```suggestion
          when jumping or falling (default: `1`)
            * Note: The actual acceleration is the product of `speed` and `acceleration_air`
```

        * 9 - zoom
    * Returns `0` (no bits set) if the object is not a player.
* `set_physics_override(override_table)`
    * Overrides the physics attributes of the player
    * `override_table` is a table with the following fields:
        * `speed`: multiplier to default movement speed and acceleration values (default: `1`)
        * `jump`: multiplier to default jump value (default: `1`)
        * `gravity`: multiplier to default gravity value (default: `1`)
        * `speed_climb`: multiplier to default climb speed value (default: `1`)
            * Note: The actual climb speed is the product of `speed` and `speed_climb`
        * `speed_crouch`: multiplier to default sneak speed value (default: `1`)
            * Note: The actual sneak speed is the product of `speed` and `speed_crouch`
        * `liquid_fluidity`: multiplier to liquid movement resistance value
          (for nodes with `liquid_move_physics`); the higher this value, the lower the
          resistance to movement. At `math.huge`, the resistance is zero and you can
          move through any liquid like air. (default: `1`)

>           Warning: Values below 1 are currently unsupported.

Reviewer 2: ```suggestion
            * Warning: Values below 1 are currently unsupported.
```

Otherwise, the line break won't be visible in the rendered version.
